packages/wallet-sdk/src/store/store.ts

```diff
@@ -143,6 +144,22 @@ export const sdkstore = createStore(
   )
 );
 
+// Non-persisted subaccount configuration
```

Comment:
I wonder if we should persist the config (excluding the toOwnerAccount function)

Reply:
we always load it from the sdk config anyway so no point really?